### DISCUSSION TITLE
Open issue on Support Repo: Add eddie emote

### DISCUSSION THREAD

**Discussion by Vyvy-vi** [EddieHubCommunity/support#529] [First-Timers] [answered]
Could someone open an issue on the `support` repo, to remind Eddie to add an emoji to the discord server.
(this is for first-timers wanting to contribute🙃 ) 🟩 

The Issue:
```
Type: Community Question
Title- Add Eddie Emoji on Discord
Description- There was an emoji request to add an emoji to the discord server, and Eddie planned on adding it. This issue is a reminder for Eddie to add the custom emoji to the discord server. 
How to add emojis to discord?:  https://www.businessinsider.com/how-to-add-emojis-to-discord?IR=T
```

CONTEXT: 
![Screenshot 2020-12-08 at 10 14 56 PM](https://user-images.githubusercontent.com/62864373/101514531-5f85f880-39a3-11eb-88ca-df6de4b8170a.png)

Emoji Creator-
@TheShubham99  (https://github.com/EddieJaoudeCommunity/support/discussions/208)

The emoji png file(after cropping and removing background)- 
https://media.discordapp.net/attachments/700356994845900852/785613003416141884/eddietest.png

**Comment by Vyvy-vi** [comment, marked as answer]
someone just opened an issue 😅 
[support#530](https://github.com/EddieJaoudeCommunity/support/issues/530)

**Reply by eddiejaoude** [reply]
Awesome 👍
